src/test/kotlin/org/rust/ide/intentions/ToggleFeatureIntentionTest.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import org.intellij.lang.annotations.Language
+import org.rust.MockCargoFeatures
+import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.project.workspace.FeatureState
+import org.rust.cargo.project.workspace.PackageOrigin
+import org.rust.singleProject
+import org.rust.workspaceOrFail
+
+class ToggleFeatureIntentionTest : RsIntentionTestBase(ToggleFeatureIntention::class) {
+    fun `test unknown feature`() = doUnavailableTest(
+        """
```

Comment:
Btw, don't rely on IDEA formatter in our tests

Reply:
Damn, not again :( I always invoke `Ctrl + Shift + I` automatically by instinct, but it breaks formatting in tests.